tools/chrome-debug-extension/README.md

```diff
@@ -22,7 +22,7 @@ Your browser will load the latest version of the tool each time you restart the
 
 ## How to run the tool
 
-Click the `INSERT_ICON_HERE` icon next to the address bar to launch the tool.
+Click the `small puzzle piece` icon next to the address bar and select `Telemetry Viewer`![Telemetry-Viewer-icon](https://github.com/microsoft/ApplicationInsights-JS/blob/ChromeDebugExtension/tools/chrome-debug-extension/images/icon-19.png) from the extension dropdown to launch the tool.
```

Comment:
We will eventually need to update this to point to the master version as ChromeDebugExtension branch will be removed once we merge

Reply:
updated to relative path as well